packages/validator/src/util/params.ts

```diff
@@ -53,8 +53,11 @@ export function assertEqualParams(localConfig: ChainConfig, externalSpecJson: Sp
     }
 
     if (key === "BLOB_SCHEDULE") {
+      // Parse JSON string if needed (interopConfigs stores BLOB_SCHEDULE as JSON string)
+      const remoteBlobScheduleValue =
+        typeof externalSpecJson[key] === "string" ? JSON.parse(externalSpecJson[key]) : externalSpecJson[key];
       const localBlobSchedule = deserializeBlobSchedule(localSpecJson[key]).sort((a, b) => a.EPOCH - b.EPOCH);
-      const remoteBlobSchedule = deserializeBlobSchedule(externalSpecJson[key]).sort((a, b) => a.EPOCH - b.EPOCH);
+      const remoteBlobSchedule = deserializeBlobSchedule(remoteBlobScheduleValue).sort((a, b) => a.EPOCH - b.EPOCH);
```

Comment:
these changes shouldn't be required, the `BLOB_SCHEDULE` should be an array, not a string